apps/desktop-wallet/src/components/NFTCard.tsx

```diff
@@ -17,139 +17,56 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { NFT } from '@alephium/shared'
-import { motion, useMotionValue, useSpring, useTransform } from 'framer-motion'
-import { useState } from 'react'
+import { colord } from 'colord'
+import { motion } from 'framer-motion'
 import styled from 'styled-components'
 
-import Card3D, { card3DHoverTransition } from '@/components/Card3D'
+import NFTThumbnail from '@/components/NFTThumbnail'
 import Truncate from '@/components/Truncate'
 
 interface NFTCardProps {
   nft: NFT
+  onClick?: () => void
 }
 
-// TODO: Copied from explorer
+const NFTCard = ({ nft, onClick }: NFTCardProps) => (
```

Comment:
It would be better to pass the `nftId` and fetch it from the store, like you do in `NFTDetailsModal` . It reduces re-renders.